source/__tests__/integration/new-profile.test.js

```diff
@@ -0,0 +1,64 @@
+describe('Adding and removing a new Profile', () => {
+
+    beforeAll(async () => {
+        await page.goto('http://localhost:8000/#/profile/create')
+    })
+
+    it('displays a new profile span', async () => {
+        const span = await page.$eval('span[id="profileBanner"] > span', e => e.getAttribute('popover-title'))
+        await expect(span).toMatch(/Profile ID: Undefined/)
+    })
+
+    describe('Adding a Resource Template', () => {
+
+        beforeAll(async () => {
+            await page.waitForSelector('#addResource')
+            await page.click('#addResource')
+        })
+
+        it('displays a new resource template span', async () => {
+            const title = await page.$eval('span[id="0"] > span', e => e.getAttribute('popover-title'))
+            const span = await page.$eval('span[id="0"] > span', e => e.textContent)
+            expect(title).toMatch(/Resource ID: Undefined/)
+            expect(span).toMatch(/Resource Template/)
+        })
+
+        it('has six input fields for the profile admin data', async () => {
+            const inputs = await page.$eval('div[id="profile"] > div > table', e => e.getElementsByTagName('input').length)
+            expect(inputs).toBe(6)
+        })
+
+        it('has five input fields for the profile admin data', async () => {
```

Comment:
it seems like the description in line 31 should be diff from line 26.  Same question about what would be a good test.

Reply:
Copy pasta error. Will fix.